packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -494,7 +494,10 @@ export class ScheduleTrigger implements INodeType {
 
 			if (interval[i].field === 'hours') {
 				const hour = interval[i].hoursInterval as number;
-				const minute = interval[i].triggerAtMinute?.toString() as string;
+				let minute = interval[i].triggerAtMinute?.toString() as string;
+				if (!minute && nodeVersion >= 1.2) {
+					minute = '0';
```

Comment:
Doesn't the same issue exist in a lot of other places in the Schedule node? You can delete any number field, and we don't set a fallback

Reply:
good point, will push update